rllib/utils/replay_buffers/episode_replay_buffer.py

```diff
@@ -537,11 +575,38 @@ def to_sample_batch(self):
                 SampleBatch.TRUNCATEDS: np.array(
                     [False] * (len(self) - 1) + [self.is_truncated]
                 ),
+                # TODO (simon): Check, if indexing is right here.
+                SampleBatch.INFOS: self.infos[:-1],
+                **self.extra_model_outputs,
             }
         )
 
     @staticmethod
     def from_sample_batch(batch):
+        # TODO (simon): This is very ugly, but right now
```

Comment:
What if we only make this list:
* OBS
* ACTIONS
* REWARDS
* TRUNCATEDS
* TERMINATEDS
* EPS_ID

Explanation:
- NEXT_OBS, PREV_REWARD|ACTION are superfluous as we are dealing with a complete episode here anyways.
- DONES are directly dependent on terminateds + truncateds.
- CUR_OBS == OBS
- SEQ_LENS should not be part of an Episode (the entire Episode is the sequence in a way)
- T: We might want to verify that the values under `T` are continuously increasing and if not, error out. If yes, we should set `Episode.t_started` to the first T in the SampleBatch, and `Episode.t` to the last T in the given SampleBatch (we might have to enhance the c'tor of Episode to be able to do that).
- AGENT_INDEX: ignore. We are dealing with single-agent Episodes thus far and agent-ids are something that a MultiAgentEpisode should be handling on a higher level.
- ENV_ID: ignore for now
- everything else, yes, should go under `extra_model_outs`: